recipes/coverage-lcov/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "coverage-lcov" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3de40d620cb13e43c746126be3154252d1296787a9f2a54b14683763cf9c0c46
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - coverage-lcov = coverage_lcov.cli:main
+    - test = pytest:main
```

Comment:
It's good to know that this was the issue but it would be better if that is not included as the test should only be internal to your project. Otherwise, you'll block the test command for everyone that installs your package. 

Your current master branch already has that so I would recommend just cutting a new release and then using that here.

Reply:
Oh, OK, I understand now. Then I'll have to request that upstream, since this is not my package.

Reply:
Ahh sorry yes just realised that it's not your fork. Somehow I thought it is.

I would just open an issue there and ask them as it currently is fixed in master.

Reply:
@TheCleric Sorry for pinging you directly but there are currently no issues enabled for coverage-lcov. Would it be possible to cut a new release as we would need the removal of `test` here?

Thanks!

Reply:
Thank you!

It's not directly related to the meta.yaml. The problem was that previously the package declared a _tool.poetry.scripts_ called `test`. This meant that when installing the package, the command `test` would launch this package which might collide with other (more) valid uses of this command.
The version currently on master doesn't have this anymore.
Since conda-forge builds the latest release and not from master, it would be nice to have that available there. 

Ref: https://github.com/TheCleric/coverage-lcov/commit/1df022cc324351108cbcad052e22c8b9ff7f35c3

Reply:
Ahh, I thought I had released latest, but obviously not!

New release of 0.2.4 created with all current code.

Reply:
Thank you!

@tdido Please update the recipe and then this should be good to go.